src/gui/gui.pri

```diff
@@ -162,12 +162,12 @@ SOURCES += \
     $$PWD/watchedfolderoptionsdialog.cpp \
     $$PWD/watchedfoldersmodel.cpp
 
-win32|macx {
+win32|macos {
     HEADERS += $$PWD/programupdater.h
     SOURCES += $$PWD/programupdater.cpp
 }
 
-unix:!macx:dbus {
+unix:dbus {
```

Comment:
I think this is wrong. `common/macx.conf` includes `common/mac.conf` which in turn includes `common/unix.conf`. The last one appends to `QMAKE_PLATFORM` the `unix` value. We don't want dbus on mac.

Reply:
This PR is ditching `macx` so what you said shouldn't happen.

Reply:
It seems macx is still determined by qmake so replacing it with `macos` doesn't seem to be meaningful now.
I should close this PR.